### ISSUE TITLE
Screen reader issues with HAX editor

### ISSUE THREAD

**Issue by JAlbert2** [elmsln/issues#536]
The Lighthouse tool alerted on ten buttons that do not have a label or name attached to their tags. This is an issue with screen readers, as they will only say "Button." While the button tag clearly has simple-icon tags, screen readers do not look for these tags within a button tag. All that is needed it a label or name tag within the button tag itself.

**Comment by btopro** [comment]
@collinkleest  good step into accessibility. This is most likely minor / something you can replicate using the WAVE plugin, identify and research how best to resolve / ask for details after making the issue appear.

**Comment by collinkleest** [comment]
Couldn't replicate in WAVE but replicated in lighthouse, look like it has to do with `a11y-media-button` which slots the simple-icon in it. The actual button tag in `a11y-media-button` should have a name but not sure where we would get this name from.

**Comment by btopro** [comment]
as in a `name` attribute you mean? we can discuss on call today
